main.go

```diff
@@ -29,45 +30,50 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(tempov1alpha1.AddToScheme(scheme))
+	utilruntime.Must(configtempov1alpha1.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
 }
 
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var configFile string
```

Comment:
The values from flags should be probably propagated to the config file.

e.g. the config file has an option to set metrics, enable leader election, health probe.

Reply:
What do you mean? If the config file specify those values it will override the flags, otherwise the flag values will be the ones that will be used.